src/Service.Tests/SqlTests/RestApiTests/Put/PutApiTestBase.cs

```diff
@@ -412,6 +412,53 @@ public virtual async Task PutOne_Nulled_Test()
                 );
         }
 
+        /// <summary>
+        /// Test to validate successful execution of a request when a computed field is missing from the request body.
+        /// In such a case, we don't attempt to NULL out computed field (as per PUT semantics) but instead skip updating/inserting the field. 
+        /// </summary>
+        [TestMethod]
+        public virtual async Task PutOneWithComputedFieldMissingFromRequestBody()
+        {
+            // Validate successful execution of a PUT update when a computed field (here 'last_sold_on_update')
+            // is missing from the request body.
+            string requestBody = @"
+            {
+                ""book_name"": ""New book"",
+                ""copies_sold"": 101
+            }";
+            string expectedLocationHeader = $"id/1";
+
+            await SetupAndRunRestApiTest(
+                    primaryKeyRoute: expectedLocationHeader,
+                    queryString: null,
+                    entityNameOrPath: _entityWithReadOnlyFields,
+                    sqlQuery: GetQuery("PutOneUpdateWithComputedFieldMissingFromRequestBody"),
+                    operationType: EntityActionOperation.Upsert,
+                    requestBody: requestBody,
+                    expectedStatusCode: HttpStatusCode.OK
+                );
+
+            // Validate successful execution of a PUT insert when a computed field (here 'last_sold_on_update')
+            // is missing from the request body.
```

Comment:
how are you validating that DAB doesn't attempt to null out the computed field `last_sold_on_update` when that column does not have a default value and is nulled to begin with? (essentially, Ani's comment on another file).

Reply:
I have updated the column `last_sold_on` to have a default value. So the values for both `last_sold_on` and `last_sold_on_date` would be non-null after insertion.